tslint.json

```diff
@@ -27,6 +27,16 @@
 		"readonly-array": true,
 		"readonly-keyword": [true, "ignore-class"],
 		"strict-boolean-expressions": false,
-		"strict-type-predicates": false
+		"strict-type-predicates": false,
+		// Better to have functionally similar items be close together.
+		"ordered-imports": false,
```

Comment:
I feel like imports can be ordered by this rule mainly for merging problem.

Reply:
We agreed in the meeting about member ordering, I wasn't sure about import ordering but I thought that the same logic applied.

Reply:
yes, but i don't think we put a lot of meaning into the import order as member ordering

Reply:
I think ordered imports are fine but not member ordering